smokes_mojo/lib/SmokesMojo/Controller/Api.pm

```diff
@@ -85,6 +88,56 @@ $msg
 EOS
 }
 
+sub post_coresmokedb_report ($self) {
+    my $req = $self->req;
+
+    my $schema = SmokeReports::Dbh->schema;
+    my $sdb = $schema->resultset("Perl5Smoke");
+    my $id = $sdb->get_column("report_id")->max;
+    $id = $id + 1;
+
+    my $json = Cpanel::JSON::XS->new->utf8;
+    my $report_data = $json->decode($req->body);
+    $report_data = { %{ delete $report_data->{'sysinfo'} } };
+    $report_data = { %{ delete $report_data->{'config'} } };
```

Comment:
Am I misunderstanding this?

It looks like you replace all of `$report_data` with the contents of `$report_data->{sysinfo}`, then you do the same with `$report_data->{config}`, but a) you've just replaced all of `$report_data` with whatever's in sysinfo, but that has no config key, and the original `$report_data` didn't have a `config` key either.

So I'm confused.

Reply:
Indeed! I updated

Reply:
There's other parts that I blindly copied from CoreSmokeDB. Also I think there's probably something to do with 'config**s**' (or report parsing is well handling it? 🤔 I don't know)